### DISCUSSION TITLE
ERROR 331: cannot open hotstart interface file %s.

### DISCUSSION THREAD

**Discussion by Yunp-J** [pyswmm/pyswmm#405] [Q&A]
Hi, i'm tring to execute the inp file through pyswmm. It goes well, but it shows an error when I set the hotstart file as follow:
![image](https://user-images.githubusercontent.com/39219239/219947859-1387a58c-9bb1-4233-88d0-891d1e032c54.png)
And the error:
![image](https://user-images.githubusercontent.com/39219239/219947909-3af77df2-1c37-416a-a457-d6121d85bd09.png)

And actually I can run the model with hsf file in SWMM 5.1. Do you have any ideas about it? Thanks a lot :)

**Comment by bemcdonnell** [comment]
@782910254, either the path you have set to your HSF inside your INP file is incorrect or the HSF is incorrectly formatted. This is a SWMM thing, not a pyswmm thing. 

I suggest you try to SAVE a new HSF to make sure the path is correct, then USE it. That way, SWMM will format the HSF correctly.
